doc/api/test.md

```diff
@@ -331,7 +331,13 @@ changes:
   does not have a name.
 * `options` {Object} Configuration options for the test. The following
   properties are supported:
-  * `concurrency` {number} The number of tests that can be run at the same time.
+  * `concurrency` {number|boolean} If a number is provided,
+    then that many tests would run in parallel.
+    If truthy, when running in `--test` mode, it would run
+    (number of cpu cores - 1) tests in parallel.
+    For subtests, it will be infinity tests in parallel.
+    If falsy, when running in `--test` mode & subtest level
+    it would only run one test at a time.
     If unspecified, subtests inherit this value from their parent.
     **Default:** `1`.
```

Comment:
```suggestion
    * `concurrency` {number|boolean} If a number is provided,
    then that many tests would run in parallel.
    If truthy, it would run (number of cpu cores - 1) 
    tests in parallel.
    For subtests, it will be `Infinity` tests in parallel.
    If falsy, it would only run one test at a time.
    If unspecified, subtests inherit this value from their parent.
    **Default:** `false`.
```
@MoLow is this okay? I think I might not be on the right track.